lightning-network-book.tex

```diff
@@ -69,9 +69,9 @@
     \url{https://en.wikipedia.org/w/index.php?title=Digital_signature&oldid=876680165}}}
 \newglossaryentry{gossip protocol}{name={Gossip Protocol},
   description={Lightning Network nodes send and recieve information about the \gls{topology} of the Lightning Network through gossip messages which are exchanged with their peers.
-    The gossip protocol is mainly defined in BOLT 7 and defines the format of the node\_announcement, channel\_announcment and channel\_updete messages.
-    In order to prevent SPAM node anouncement messages will only be forwarded if the node already has a channel and channel anounce messages will only be forwarded if the funding transaction of the channel has been confirmed by the bitcoin network.
-    Usually Lightning Nodes peer with their channel partners but it is fine to peer with any other Lightning node in order to process gossip messages.}}
+    The gossip protocol is mainly defined in BOLT 7 and defines the format of the node\_announcement, channel\_announcement and channel\_update messages.
+    In order to prevent SPAM, node announcement messages will only be forwarded if the node already has a channel and channel announcement messages will only be forwarded if the funding transaction of the channel has been confirmed by the Bitcoin network.
+    Usually Lightning Nodes peer with their channel partners, but it is fine to peer with any other Lightning node in order to process gossip messages. \todo{I'm not familiar with 'peer' being used as a verb in this context (but maybe it's normal in networking?). Could replace with 'connect'. - BG}}}
```

Comment:
connect is the way to go. that also goes along with the api where people connect with a peer